core/src/main/java/com/digitalpebble/storm/crawler/spout/RandomURLSpout.java

```diff
@@ -42,9 +42,7 @@
     private boolean active = true;
     private boolean removeAfterSending = true;
 
-    private String[] urls = new String[] { "http://www.lequipe.fr/",
-            "http://www.lemonde.fr/", "http://www.bbc.co.uk/",
-            "http://www.facebook.com/", "http://www.rmc.fr" };
+    private String[] urls = new String[] { "http://firefoxmania.uci.cu/"};
```

Comment:
Yep this slipped through my fingers, I modified to avoid the use a proxy to do the tests, I'll fix this.